src/flask_sqlalchemy/__init__.py

```diff
@@ -172,7 +172,14 @@ def __init__(self, db, autocommit=False, autoflush=True, **options):
             **options,
         )
 
-    def get_bind(self, mapper=None, **kwargs):
+    def get_bind(
+        self,
+        mapper=None,
+        clause=None,
+        bind=None,
+        _sa_skip_events=None,
+        _sa_skip_for_implicit_returning=False,
```

Comment:
```suggestion
    def get_bind(
        self,
        mapper=None,
        *args,
        **kwargs,
```

Reply:
And they use `super().get_bind(*args, mapper=mapper, **kwargs)`

Reply:
Hmm, I made that work, but now I'm getting "multiple values for mapper"

Reply:
Oh,
 `super().get_bind(mapper, *args, **kwargs)`

Reply:
^^ @dpgaspar

Reply:
```
TypeError: get_bind() got multiple values for argument 'mapper'
```

Reply:
`mapper, *args` vs `*args, mapper` I think (I edited my second comment)